.github/workflows/test.yaml

```diff
@@ -0,0 +1,54 @@
+name: Test Volatility3
+on: [push]
+jobs:
+
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up Python 3.x
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
```

Comment:
Volatility 3 recommends `3.6` or higher, so shouldn't we adjust that version to `3.6` as well?

Reply:
I agree with this, fixed in https://github.com/volatilityfoundation/volatility3/pull/789/commits/f96f004b13c15b3c0334f7ca96b2c0459a8f9cf1